pkg/tracing/wire/binary.proto

```diff
@@ -1,44 +1,43 @@
 syntax = "proto3";
 package wire;
+option go_package = ".;wire";
 
-import "gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
 
 message SpanContext {
-  uint64 trace_id = 1 [(gogoproto.customname) = "TraceID"];
-  uint64 span_id = 2 [(gogoproto.customname) = "SpanID"];
+  uint64 TraceID = 1;
+  uint64 SpanID = 2;
 }
 
 message Span {
-  SpanContext context = 1 [(gogoproto.nullable) = false];
```

Comment:
The places we have dropped `nullable = false` worry me, because now any time we read these types we have to be sure they are not null.

I think the best would be to check that we always use the accessors (like `foo.GetSpanContext().GetTraceID()` instead of `foo.SpanContext.TraceID`) - @DStrand1 any thoughts on how we can ensure this is safe?

Reply:
Using accessors should be safe, as they always null check before attempting to access. I had issues with this for the TimestampRange Start and End before, so doing this for more fields is a good idea